apps/api/src/controllers/v2/types.ts

```diff
@@ -293,6 +293,10 @@ const baseScrapeOptions = z
     mobile: z.boolean().default(false),
     parsers: parsersSchema.optional(),
     actions: actionsSchema.optional(),
+    extractDataAttributes: z.array(z.object({
```

Comment:
Should not be a separate property on its own -- should be in `formats` instead. See JSON for example, let's call it `attributes` since it doesn't have to be a `data-` to work, and could be useful for other things too.